addons/website_sale/models/product.py

```diff
@@ -317,6 +317,19 @@ def _create_first_product_variant(self, log_warning=False):
         """
         return self._create_product_variant(self._get_first_possible_combination(), log_warning)
 
+    def _get_image_owner(self):
+        """Returns the owner of the image to use as default representation.
+        If the product template has an image it is the product template,
+        otherwise if the product has variants it is the first variant
+        
+        :return: this product template or the first product variant
+        :rtype: recordset of 'product.template' or recordset of 'product.product'
+        """
+        self.ensure_one()
+        if self.image_1920 or len(self.product_variant_ids) < 2:
+            return self
+        return self._create_first_product_variant()
```

Comment:
in short
_get_first_possible_variant_id seems less obscure
 
< 2  or >= 2 ?

_get_image_owner ? 

+ write test